docs/buildah-manifest-inspect.1.md

```diff
@@ -16,6 +16,17 @@ Displays the manifest list or image index stored using the specified image name.
 
 A formatted JSON representation of the manifest list or image index.
 
+## OPTIONS
+
+**--authfile** *path*
+
+Path of the authentication file. Default is ${XDG\_RUNTIME\_DIR}/containers/auth.json, which is set using `buildah login`.
+If the authorization state is not found there, $HOME/.docker/config.json is checked, which is set using `docker login`.
```

Comment:
Risk of divergence:  I suggest pointing to `containers-auth.json` instead

Reply:
I have copied it from other buildah docs and it matches with the text here: https://docs.podman.io/en/latest/markdown/podman-login.1.html#authfile-path

Could you suggest what needs to be changed here.

Reply:
Ah OK, then let's stop me from derailing this PR into fixing docs :)